edx_sga/sga.py

```diff
@@ -70,7 +70,7 @@ class StaffGradedAssignmentXBlock(XBlock):
         scope=Scope.settings
     )
 
-    points = Float(
+    points = Integer(
         display_name="Maximum score",
         help=("Maximum grade score given to assignment by staff."),
         values={"min": 0, "step": .1},
```

Comment:
I played around with this quite a bit. It looks like there are no actual data guards here, and the values dictionary doesn't appear to do anything.  Even with the min set at 0, it happily lets you set -50 as the value.  I'll go ahead and remove the whole parameter, but I'm not sure what to do to do integrity checks beyond it being a whole number.